src/Components/Recommendations/getLocalStorage.js

```diff
@@ -0,0 +1,12 @@
+export default function GetLocalStorage(bookPosition) {
```

Comment:
Where is this function used? I think it's just used in the BookPage, so once the carousels are added it might be better to move this there! Don't worry about it for this sprint though, this is a note for whoever styles the bookPage in the end.